client/rest/build.gradle

```diff
@@ -78,7 +78,8 @@ task shadeDeps(type: com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar)
   destinationDir = shadedDir
   configurations = [project.configurations.shade]
   classifier = 'deps'
-  relocate 'org.apache', 'org.elasticsearch.client'
+  relocate 'org.apache.commons', 'org.elasticsearch.client.commons'
+  relocate 'org.apache.http', 'org.elasticsearch.client.http'
```

Comment:
I do not think we should do this. I think we should shade nothing (my vote) or everything, no partial shading. I also do not see a clear consensus having been reached either way.